docs/user/rstudio/ide/guide/tools/jobs.qmd

```diff
@@ -2,98 +2,107 @@
+## Showing task progress
 
-![](images/workbench-job-progress.png){fig-alt="IDE tab showing status of Workbench jobs" width="550"}
+RStudio's Jobs pane can show more than just the progress of Background or Workbench jobs. It can also be scripted from R packages (and R code) to show status, progress, and output for any long-running task.
 
-You can also monitor the status and progress of your Workbench Jobs on your Posit Workbench dashboard:
+To show progress and/or output from a task using the jobs UI, refer to the [rstudioapi documentation](https://rstudio.github.io/rstudioapi/reference/index.html#jobs) for details; start with `jobAdd()`, which creates a new job in the UI and returns a handle you can use to update the UI as the job progresses.
 
-![](images/workbench-homepage-progress.png){fig-alt="Posit Workbench dashboard showing executing jobs" width="450"}
+| Interact with the jobs pane.                                                             |     |
+|:------------------------------|-----------------------------------------|
+| [`jobAdd()`](https://rstudio.github.io/rstudioapi/reference/jobAdd.html)                 |   Add a Job  |
+| [`jobAddOutput()`](https://rstudio.github.io/rstudioapi/reference/jobAddOutput.html)     |   Add Background Job Output  |
+| [`jobAddProgress()`](https://rstudio.github.io/rstudioapi/reference/jobAddProgress.html) |  Add Background Job Progress   |
+| [`jobRemove()`](https://rstudio.github.io/rstudioapi/reference/jobRemove.html)           |   Remove a Background Job  |
+| [`jobSetProgress()`](https://rstudio.github.io/rstudioapi/reference/jobSetProgress.html) |  Set Background Job Progress   |
+| [`jobSetState()`](https://rstudio.github.io/rstudioapi/reference/jobSetState.html)       |  Set Background Job State   |
+| [`jobSetStatus()`](https://rstudio.github.io/rstudioapi/reference/jobSetStatus.html)     |   Set Background Job Status  |
 
-## Showing task progress
+## Workbench (Launcher) jobs {.pro-header}
 
-RStudio's Jobs pane can show more than just the progress of Background or Workbench jobs. It can also be scripted from R packages (and R code) to show status, progress, and output for any long-running task.
+On Posit Workbench, you also have the option of running your R script on your company's compute infrastructure, using the [Job Launcher](https://blog.rstudio.com/2018/11/05/rstudio-rsp-1.2-features/). To do this, select:
 
-If you'd like to show progress and/or output from a task using the jobs UI, refer to the [rstudioapi documentation](https://rstudio.github.io/rstudioapi/reference/index.html#jobs) for details; start with `addJob`, which creates a new job in the UI and returns a handle you can use to update the UI as the job progresses.
+![](images/workbench-job.png){fig-alt="Source script as launcher job" width="350"}
 
-Copied from <https://solutions.rstudio.com/r/jobs/>
+See more documentation specific to Workbench Launcher Jobs at <https://docs.posit.co/ide/server-pro/user>
```

Comment:
This link doesn't open for me

Reply:
This is a future link - for now I've moved it back to <https://docs.rstudio.com/ide/server-pro/> while we wait for the Workbench User Guide to be finalized.